src/app/games/lasca/LascaTutorial.ts

```diff
@@ -0,0 +1,78 @@
+import { Tutorial, TutorialStep } from 'src/app/components/wrapper-components/tutorial-game-wrapper/TutorialStep';
+import { Coord } from 'src/app/jscaip/Coord';
+import { LascaMove } from './LascaMove';
+import { LascaPiece, LascaSpace, LascaState } from './LascaState';
+
+const zero: LascaPiece = LascaPiece.ZERO;
+const one: LascaPiece = LascaPiece.ONE;
+const _u: LascaSpace = new LascaSpace([zero]);
+const _v: LascaSpace = new LascaSpace([one]);
+const __: LascaSpace = LascaSpace.EMPTY;
+
+export class LascaTutorial extends Tutorial {
+    public tutorial: TutorialStep[] = [
+        TutorialStep.informational(
+            $localize`Lasca: origins`,
+            $localize`Lasca is a game based on draughts created in 1911 by Emanuel Lasker, chess world champion. It's played on a 7x7 board, each player has 11 pieces.`,
```

Comment:
je pensais déjà avoir commenté : ok c'est informationnel, mais décrire ce qui est 1000x mieux décrit par une image, visualisé à droite dans le tuto, c'est pas très utile non ? Genre je vois que c'est un 7x7, et concrètement je m'en branle que chacun ait 11 pièces ou 8, mais je le vois. Dire « c'est joué sur le plateau représenté ici dans sa configuration initiale » ou du genre est tout autant informationnel mais plus concret

Reply:
Être explicite aide certaine personne, même si c'est choquant de devoir expliquer ce qu'est une case et une pièce :D mais une étape de description du plateau, du lexique, c'est pas mal ! Après c'est pas aussi nécessaire pour tout les plateaux mais présenter le plateau c'est pas mal.

Reply:
faut voir si on souhaite que les tutos soient une page wikipédia ou quelque chose qui sert aux gens à savoir jouer en 5 minutes ! Si y'a de l'information superflue dans le texte, ça va pas aider à ce qu'ils le lisent :p

Reply:
Mais en même temps ça, c'est l'intro quoi, on racontes pas la vie de Mr. Lasker au moment ou on explique les règles et les how-to :D